common/djangoapps/course_modes/models.py

```diff
@@ -608,6 +609,9 @@ def is_white_label(cls, course_id, modes_dict=None):
         if modes_dict is None:
             modes_dict = cls.modes_for_course_dict(course_id)
 
+        if configuration_helpers.is_site_configuration_enabled():
+            return configuration_helpers.get_value('show_ecommerce_reports', False)
```

Comment:
Are White label courses based on 'PROFESSIONAL' and 'NO_ID_PROFESSIONAL' ? then we can do something like that ? 
```
if configuration_helpers.is_site_configuration_enabled() and \
                any((True for _mode in modes_dict if _mode in [cls.PROFESSIONAL, cls.NO_ID_PROFESSIONAL_MODE])):
```